doc/installation.rst

```diff
@@ -1,23 +1,48 @@
 Installation
 ============
 
+.. _requirements:
+
+Requirements
+------------
+The random forest used in SMAC depends on a c++11 compatible compiler
+and on SWIG (>3.0).
+
+To install both requirements system-wide on a linux system with apt, 
+please call:
+
+.. code-block:: bash
+
+    sudo apt-get install gcc swig
```

Comment:
That's what I recall as well. I installs `make` as well, so I think it would make sense to change this.